brotab/api.py

```diff
@@ -97,7 +97,7 @@ def activate_tab(self, args):
         #self._get('/activate_tab/%s' % strWindowTab)
 
     def get_active_tab(self, args) -> str:
```

Comment:
I think name and the type should be updated as well

Reply:
oof. I missed the annotation and thought the `_tabs` suffix was taken by another function but in fact it is a member of the sister class.